servlet-engine/src/main/java/io/micronaut/servlet/engine/bind/DefaultServletBinderRegistry.java

```diff
@@ -108,24 +110,25 @@ public BindingResult bind(ArgumentConversionContext context, HttpRequest source)
                 Class<?> javaArgument = typeArgument.getType();
                 Charset characterEncoding = servletHttpRequest.getCharacterEncoding();
                 if (CharSequence.class.isAssignableFrom(javaArgument)) {
-                    Flowable.fromPublisher(servletHttpRequest).collect(StringBuilder::new, (stringBuilder, bytes) ->
+                    Flux.from(servletHttpRequest).collect(StringBuilder::new, (stringBuilder, bytes) ->
                             stringBuilder.append(new String(bytes, characterEncoding))
-                    ).subscribe((stringBuilder, throwable) -> {
```

Comment:
I think there is method which takes both consumer of 
```
subscribe(Consumer<? super T> consumer,
          Consumer<? super Throwable> errorConsumer);
```

Reply:
I have already switched to that.